### PR TITLE
Taxonomy terms ncbi migration

### PR DESCRIPTION
**Pull request by CarlosHorro**
Migration file and csv file with NCBI - GR mapping.

### CODE REVIEW

**Comment by Nuanda** [comment]
@CarlosHorro Carlos, @kammerer will review this after he's back from his leave -- beginning of October.

**Comment by kammerer** [comment]
@CarlosHorro In the future you can make small PRs as such directly against master branch.

File: db/schema.rb [hunk 1]
```diff
@@ -57,6 +57,7 @@
   end
 
   add_index "api_keys", ["token"], name: "api_keys_token_idx", unique: true, using: :btree
+  add_index "api_keys", ["token"], name: "index_api_keys_on_token", using: :btree
```
Comment (kammerer) [inline db/schema.rb]: Please, do not commit any changes to `db/schema.rb` which are not related to the migration you added.

File: db/taxonomy_terms_ncbi_updated.csv [hunk 2]
```diff
@@ -0,0 +1,189 @@
+ID,LABEL,New_Label,ncbi_taxon,NAME,TAXONOMY_TERM_ID,CREATED_AT,UPDATE_AT,CANONICAL
```
Comment (kammerer) [inline db/taxonomy_terms_ncbi_updated.csv]: `db/data/taxonomy_terms_ncbi_updated.csv` would be a slightly better location.

File: db/schema.rb (hunk 1 above)
Comment (CarlosHorro) [reply db/schema.rb]: Sorry, I don't know why these changes were generated automatically.

**Pull request merged**